### PR TITLE
Update to binja 3.5

### LINKED ISSUE #1789: Fix unit test after Binary Ninja 3.5 release
We have released Binary Ninja 3.5 last Friday. I thought we simply need to update the binja version check here: https://github.com/mandiant/capa/blob/master/tests/test_binja_features.py#L72, since it explicitly checks for version 3.4. 

However, as it seems, the failures is more deadly than expected: it seems BN 3.5 crashes during the unit test: https://github.com/mandiant/capa/actions/runs/6235434766/job/16924696404#step:7:70. I will have a look at what have changed and see how it can be fixed

### PR DESCRIPTION
**Pull request by xusheng6**
### Checklist

- [ ] No CHANGELOG update needed

- [x] No new tests needed

- [x] No documentation update needed

### CODE REVIEW

**Comment by xusheng6** [comment]
@williballenthin would you like to have a look at this when you have time? These are some minor changes since we have relreased binary ninja 3.5

File: capa/main.py [hunk 1]
```diff
@@ -558,7 +558,7 @@ def get_extractor(
                 sys.path.append(str(bn_api))
 
         try:
-            from binaryninja import BinaryView, BinaryViewType
+            from binaryninja import BinaryView, load
```
Comment (williballenthin) [inline capa/main.py:561]:
```suggestion
            import binaryninja
            from binaryninja import BinaryView
```

File: tests/fixtures.py [hunk 2]
```diff
@@ -159,7 +159,7 @@ def get_dnfile_extractor(path: Path):
 
 @lru_cache(maxsize=1)
 def get_binja_extractor(path: Path):
-    from binaryninja import Settings, BinaryViewType
+    from binaryninja import Settings, load
```
Comment (williballenthin) [inline tests/fixtures.py:162]:
```suggestion
    import binaryninja
    from binaryninja import Settings
```

File: tests/fixtures.py [hunk 3]
```diff
@@ -168,7 +168,7 @@ def get_binja_extractor(path: Path):
     if path.name.endswith("kernel32-64.dll_"):
         old_pdb = settings.get_bool("pdb.loadGlobalSymbols")
         settings.set_bool("pdb.loadGlobalSymbols", False)
-    bv = BinaryViewType.get_view_of_file(str(path))
+    bv = load(str(path))
```
Comment (williballenthin) [inline tests/fixtures.py:171]:
```suggestion
    bv = binaryninja.load(str(path))
```

**Review by williballenthin — APPROVED** [review]
logic updates look great! a couple tweaks to imports that you can merge easily and then i'll merge the PR. 

thanks @xusheng6!

**Comment by xusheng6** [comment]
> logic updates look great! a couple tweaks to imports that you can merge easily and then i'll merge the PR.
> 
> thanks @xusheng6!

Thank for the suggestions! I have edited the code accordingly

**Comment by xusheng6** [comment]
@williballenthin I think you can close this issue: https://github.com/mandiant/capa/issues/1789

**Pull request merged**